src/agent/application.hpp

```diff
@@ -34,6 +34,8 @@
 #ifndef OTBR_AGENT_APPLICATION_HPP_
 #define OTBR_AGENT_APPLICATION_HPP_
 
+#include "openthread-br/config.h"
```

Comment:
The config.h header is empty as of now. What's the use case of it?

Reply:
What about removing the macro `OTBR_CONFIG_FILE`? We may not have to keep it unless there's anybody using it.

Reply:
actually it's used on Android platform.